data_scraper/__main__.py

```diff
@@ -21,6 +21,9 @@
     help="Aggregate daily data files")
 parser.add_argument(
     "-b", "--backup", action="store_true", help="Backup files in S3 bucket")
+parser.add_argument(
+    "-d","--backup_daily", action="store_true", help="Backup daily files in S3 bucket"
```

Comment:
Let's make the long option name `--backup-daily`, following the convention in the [POSIX](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) standard.